### PR TITLE
export functions to get llm clients

### PR DESCRIPTION
**Pull request by abubakarsohail**

### CODE REVIEW

**Comment by Jped** [comment]
@abubakarsohail  does this edit mean people will have to do 
`from promptlayer import get_openai` ?

**Review by hankqqu — APPROVED** [review]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Comment by hankqqu** [comment]
> def fibonacci(n)
>   return n if ( 0..1 ).include? n
>   (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
> end

**Comment by abubakarsohail** [comment]
> @abubakarsohail  does this edit mean people will have to do 
> 
> `from promptlayer import get_openai` ? 

Yes. And when the user calls this function, it will throw `ModuleNotFoundError` automatically.

File: promptlayer/__init__.py [hunk 1]
```diff
@@ -8,31 +8,24 @@
...
-        provider_type="anthropic",
-    )
-except ImportError:
-    print(
-        "Anthropic module not found. Install with `pip install anthropic`.",
-        file=sys.stderr,
-    )
-    pass
+
+def __getattr__(name: Union[Literal["openai"], Literal["anthropic"]]):
```
Comment (abubakarsohail) [inline promptlayer/__init__.py:12]: @Jped this should allow us to do lazy module initialization.

Comment (Jped) [reply promptlayer/__init__.py:12]: Let me pull this and test.

**Pull request merged**